readline/codes.go

```diff
@@ -51,8 +51,13 @@ var (
 	seqDelete2      = string([]byte{27, 91, 80})
 	seqCtrlDelete   = string([]byte{27, 91, 51, 59, 53, 126})
 	seqCtrlDelete2  = string([]byte{27, 91, 77})
+	seqAltDelete    = string([]byte{27, 91, 51, 59, 51, 126})
+	seqAltDelete2   = string([]byte{27, 91, 80}) // idk why this doesnt work in st
```

Comment:
any reason for this to be here besides st? if it doesnt work then just remove it

Reply:
can remove

Reply:
removed